lib/dialects/mysql/query.js

```diff
@@ -29,17 +29,17 @@ Query.prototype.run = function(sql, parameters) {
   //do we need benchmark for this query execution
   var benchmark = this.sequelize.options.benchmark || this.options.benchmark;
 
-  if (!benchmark) {
-    this.sequelize.log('Executing (' + (this.connection.uuid || 'default') + '): ' + this.sql, this.options);
-  } else {
+  if (benchmark) {
     var queryBegin = Date.now();
+  } else {
+    this.sequelize.log('Executing (' + (this.connection.uuid || 'default') + '): ' + this.sql, this.options);
   }
 
   var promise = new Utils.Promise(function(resolve, reject) {
     self.connection.query(self.sql, function(err, results) {
 
       if (benchmark) {
-        self.sequelize.log('Executed (' + (self.connection.uuid || 'default') + '): ' + self.sql + ' in ' + (Date.now() - queryBegin) + 'ms', self.options);
+        self.sequelize.log('Executed (' + (self.connection.uuid || 'default') + '): ' + self.sql, (Date.now() - queryBegin), self.options);
```

Comment:
Please look at Sequalize.log function: https://github.com/sequelize/sequelize/blob/master/lib/sequelize.js#L1282
If options argument is received, then it is cut out (by slice) from args that are passed to the logger function. So the user will never receive this options.